src/key_id_managers/mod.rs

```diff
@@ -20,10 +20,22 @@
 
 use crate::authenticators::ApplicationName;
 use parsec_interface::requests::ProviderID;
+use serde::Deserialize;
 use std::fmt;
 
 pub mod on_disk_manager;
 
+#[derive(Deserialize)]
+pub enum KeyIdManagerType {
+    OnDisk,
+}
+
+#[derive(Deserialize)]
+pub struct KeyIdManagerConfig {
+    pub manager_type: KeyIdManagerType,
+    pub store_path: Option<String>,
```

Comment:
I was thinking about that as well - I think so, unless it's an in-memory store in which case you can leave it empty. Otherwise, you have to store them _somewhere_

Reply:
We can leave that for now and expand later if needed!